### ISSUE TITLE
world's chilliest mangrove (coordinates need inspection)

### ISSUE THREAD

**Issue by sformel-usgs** [gbif/portal-feedback#5480]
_TL;DR Coordinates are incorrect and are being interpreted differently by GBIF and OBIS._

GBIF: https://www.gbif.org/occurrence/1090173320
OBIS: https://obis.org/occurrence/9ac63d44-af8a-4e84-8491-e655b917ded2

This mangrove is described as occurring in Palau, but the latitude is 79.3903 with a verbatim value of "79°23'25 N". I'm guessing it's supposed to be ~ 7° N.

Also worth pointing out that the OBIS mapper and GBIF map interpret these coordinates differently.  [OBIS places it correctly over eastern Russia](https://mapper.obis.org/?taxonid=235091&geometry=POLYGON%20((129.7266%2080.5897,%20130.0781%2077.7676,%20140.9766%2077.9890,%20139.9219%2080.5897,%20129.7266%2080.5897))#), while [GBIF places it incorrectly over western Canada](https://www.gbif.org/occurrence/1090173320).

 @dbloom would you mind reaching out to the museum about this? Thanks to @mathewbiddle for identifying this chilly mangrove.

**Comment by ManonGros** [comment]
Google maps plots the coordinates above Canada:
<img width="1500" alt="Screenshot 2024-09-18 at 16 01 34" src="https://github.com/user-attachments/assets/3468ab16-2fd7-4f13-a252-7014982279b1">
How do you know that they should be over Russia?

**Comment by ManonGros** [comment]
Does OBIS interpret the verbatim coordinates directly?

**Comment by dbloom** [comment]
@sformel-usgs I will alert the Bishop about this one.

**Comment by sformel-usgs** [comment]
@ManonGros the plot thickens!  `verbatimCoordinates` are `7°23'25.0"N 134°31'10.0"E`, which is over Russia (and Palau when you adjust the Latitude to 7N.) However, the `decimalLongitude`  value is negative which the equivalent to 134°31'10.0"**W** instead of 134°31'10.0"**E**

#### The plot thickens more!!

The `decimalLongitude` value in the source value in OBIS is the correct, positive `decimalLongitude` and the GBIF 'original' value is negative. This is because they are being published from two different endpoints.  OBIS is harvesting https://nzobisipt.niwa.co.nz/resource?r=bernicepbishopmuseum and GBIF is harvesting https://ipt.vertnet.org/archive.do?r=bpbm.

It looks like the SW Pacific OBIS Node is publishing the occurrence on OBIS and VertNet is publishing it on GBIF.  It's strange because the SW Pacific OBIS Node has other datasets on GBIF, but not this one.  I don't have a github handle for Kevin McKay, the OBIS node manager.  Before I write him an email:

@pieterprovoost do you have any thoughts about this? 
@dbloom the dataset from SW Pacific OBIS node appears to be a subset of what you're serving from VertNet.  Could you see if Richard Pyle knows where these are diverging?
